tests/test_connection.py

```diff
@@ -0,0 +1,21 @@
+import os
+import pprint
+from typing import Optional
+
+from jira_access.jira_query_executor import JiraQueryExecutor
+
+JIRA_BASE_URL = "https://scewosw.atlassian.net/"
```

Comment:
Do you really want to push this line?
```suggestion
JIRA_BASE_URL = Optional[str] = os.getenv("JIRA_BASE_URL ")
```

Reply:
No, it is best if we use a Mock in the test anyway, but for the moment I change it :).